tests/e2e-instrumentation/instrumentation-dotnet-musl/00-install-instrumentation.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: opentelemetry.io/v1alpha1
+kind: Instrumentation
+metadata:
+  name: dotnet
+spec:
+  env:
+    - name: OTEL_TRACES_EXPORTER
+      value: otlp
+    - name: OTEL_EXPORTER_OTLP_ENDPOINT
+      value: http://localhost:4317
+    - name: OTEL_EXPORTER_OTLP_TIMEOUT
+      value: "20"
+    - name: OTEL_TRACES_SAMPLER
+      value: parentbased_traceidratio
+    - name: OTEL_TRACES_SAMPLER_ARG
+      value: "0.85"
+    - name: SPLUNK_TRACE_RESPONSE_HEADER_ENABLED
+      value: "true"
+  exporter:
+    endpoint: http://localhost:4317
+  propagators:
+    - jaeger
+    - b3multi
+  sampler:
+    type: parentbased_traceidratio
+    argument: "0.25"
```

Comment:
I think we can just put this in the `00-install-collector.yaml` file and rename it to just `00-install.yaml.`. I don't see much benefit in making them separate.

Reply:
Files for this tests are copied from `instrumentation-dotnet`. What is more other e2e follows same pattern.

BTW I like it as a separate step - it ilustrate what you should to execute otel-operator.
I would consider to create separate issue to fix all this files.